beacon-chain/db/filters/filter.go

```diff
@@ -14,7 +14,10 @@
 //	}
 package filters
 
-import primitives "github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
+import (
+	"github.com/pkg/errors"
+	primitives "github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
```

Comment:
interesting, do we need `primitives` alias here?

Reply:
Not sure why it was aliased, IDE just preserved that when adding the new import. I'll snip it. :scissors: